xsgen/openmc_origen.py

```diff
@@ -151,15 +152,16 @@ def __init__(self, rc):
         self.eafds = data_source.EAFDataSource()
         self.omcds = data_source.OpenMCDataSource(
                         cross_sections=rc.openmc_cross_sections,
-                        src_group_struct=np.logspace(1, -9, 1001))
+                        src_group_struct=rc.openmc_group_struct)
         data_sources = [self.omcds]
-        if not rc.is_thermal:
-            data_sources.append(self.eafds)
+        #if not rc.is_thermal:
+        #    data_sources.append(self.eafds)
         data_sources += [data_source.SimpleDataSource(),
                          data_source.NullDataSource()]
         for ds in data_sources[1:]:
             ds.load()
         self.xscache = XSCache(data_sources=data_sources)
+        self.xscache.load(293.6)
```

Comment:
Yeah, I think I'll shift this over to the input file. I didn't actually hard code the temp in, John did.